perun-base/src/main/resources/perun-roles.yml

```diff
@@ -5095,50 +5103,71 @@ perun_policies:
     policy_roles: []
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: User
 
   owner-setSpecificUser_User_SpecificUserType_User_policy:
     policy_roles: []
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: User
 
   unsetSpecificUser_User_SpecificUserType_policy:
     policy_roles: []
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: User
 
   deleteUser_User_policy:
     policy_roles: []
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: User
+      - MFA: Member
```

Comment:
I think Member is affected only in the forceDelete version of this method (so the policy after this one). Also, user external sources are deleted in this method, so MFA: UserExtSource should be here imo. What do you think @Johaney-s @cuadradek ?